### ISSUE TITLE
Loss Value on VOC dataset

### ISSUE THREAD

**Issue by desmanda** [qqwweee/keras-yolo3#167]
Hi,
Can anyone please tell me about the loss value, you got when training on VOC dataset.
This will save me a day of work to debug my case.
Thanks!

**Comment by QuntuamLoop** [comment]
how about you, did you using the pretrained data and frozen the front layers?
![shot](https://user-images.githubusercontent.com/32284800/42505993-2254f1e2-8473-11e8-9a98-cde1043517eb.jpg)

**Comment by Hello526** [comment]
Hi,
Do you know the mAP of YOLOV3 on PASCAL VOC2012? I read the original text and did not mention this.
Thanks a lot!
